config/peerpods/podvm/osc-podvm-create-job.yaml

```diff
@@ -13,9 +13,11 @@ spec:
     spec:
       # Add initContainers to pull the image from the registry and copy
       # /podvm-binaries.tar.gz /payload/podvm-binaries.tar.gz
+      imagePullSecrets:
+        - name: regcred
       initContainers:
         - name: copy
-          image: registry.redhat.io/openshift-sandboxed-containers/osc-podvm-payload-rhel9:1.7.0
+          image: quay.io/suprit_iti/openshift-sandboxed-containers/osc-podvm-payload-rhel9:latest
```

Comment:
Change the image source to registry.redhat.io and use it once the changes related to Dockerfile is approved